opm/parser/eclipse/EclipseState/IOConfig/RestartConfig.cpp

```diff
@@ -195,6 +205,118 @@ namespace Opm {
     }
 
 
+    void RestartConfig::update( size_t step, const RestartSchedule& rs) {
+        if (step == 0)
+            this->restart_schedule->updateInitial( rs );
+        else
+            this->restart_schedule->update( step, rs );
+    }
+
+
+    const std::vector<std::string> RestartConfig::integerKeywords = {""     ,      //  1 BASIC=?? - ignored
```

Comment:
Why is this a member in the first place?

Unless there's a very good reason for it, this should probably be `static constexpr const char* integerKeywords[] = { ... }.` (or anon namespace as you see fit).

Why? Well, first of all it's a simpler data type for what is static (as in binary region) in the binary anyway, and you use indexing for lookup by design. Second, it opens up a few optimisations for no syntactic cost.